angular-route/templates/_.client.route.js

```diff
@@ -0,0 +1,5 @@
+$stateProvider.
+  state('<%= slugifiedName %>', {
+    url: '/<%= slugifiedRoutePath %>',
+    templateUrl: 'modules/<%= slugifiedModuleName %>/client/views/<%= slugifiedViewName %>.client.view.html'
+  }).
```

Comment:
@UndeadBaneGitHub @codydaig shouldn't any code be based on our using ui-router and not ng-route? I don't think we are using ng-route at all...